packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1032,6 +1034,11 @@ export class ContainerRuntime
 	 */
 	private nextSummaryNumber: number;
 
+	/**
+	 * If true, loading or using a Tombstoned object should merely log, not fail
+	 */
+	public readonly disableGcTombstoneEnforcement: boolean;
```

Comment:
Yeah I'm pondering this too.  Need to "get it right" because I can't rename it later (breaking change).  "Enable" sounds too authoritative, since other checks have to pass too. But "allow" may be clearer.